errors.go

```diff
@@ -196,3 +196,28 @@ type RequestErrCASWriteUnknown struct {
 	Received    int
 	BlockFor    int
 }
+
+type opType uint8
+
+const (
+	opTypeRead  opType = 0
+	opTypeWrite opType = 1
+)
```

Comment:
In Go, identifiers starting with a lowercase letters are private. I think those (opType, opTypeRead, opTypeWrite) should be public.